python/gui/symbology-ng/qgssymbolv2selectordialog.sip

```diff
@@ -64,7 +64,6 @@ class QgsSymbolV2SelectorDialog : QDialog
     void addLayer();
     void removeLayer();
 
-    void saveSymbol();
```

Comment:
You'll have to also leave this method here for now, as otherwise it's breaking the stable API. Mark it as deprecated with /Deprecated/ and Q_DECL_DEPRECATED in the header.